android/app/src/main/java/com/microsoft/codepush/react/CodePushPackage.java

```diff
@@ -18,18 +18,6 @@
 import java.nio.ByteBuffer;
 
 public class CodePushPackage {
```

Comment:
In the UWP implementation, I believe we renamed this to "UpdateManager". I'd be in favor of changing this to CodePushUpdateManager as well.